### PR TITLE
Add potato annotation tool

### PR DESCRIPTION
**Pull request by Jiaxin-Pei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Slack](https://zenml.io/slack-invite/) and leave us a message!
 | [Hover](https://github.com/phurwicz/hover) | Explore and label on a map of raw data. Handles text, audio and images. ![](https://img.shields.io/github/stars/phurwicz/hover?color=black&label=%E2%AD%90%EF%B8%8F&style=plastic) | MIT |
 | [Label Studio](https://github.com/heartexlabs/label-studio) | A multi-type data labeling and annotation tool with standardized output format ![](https://img.shields.io/github/stars/heartexlabs/label-studio?color=black&label=%E2%AD%90%EF%B8%8F&style=plastic) | Apache-2 |
 | [Pigeon](https://github.com/agermanidis/pigeon) | A simple widget that lets you quickly annotate a dataset of unlabeled examples from the comfort of your Jupyter notebook ![](https://img.shields.io/github/stars/agermanidis/pigeon?color=black&label=%E2%AD%90%EF%B8%8F&style=plastic) | Apache-2 |
+| [Potato](https://github.com/davidjurgens/potato) | An annotation framework with 20+ tempaltes, editable UI, quality control, data management and even add a surveyflow for crowdourcing ![](https://img.shields.io/github/stars/davidjurgens/potato?color=black&label=%E2%AD%90%EF%B8%8F&style=plastic) | PolyForm Shield |
```
Comment (strickvl) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -65,6 +66,7 @@ Slack](https://zenml.io/slack-invite/) and leave us a message!
 | [knodle](https://github.com/knodle/knodle) | Knodle (Knowledge-supervised Deep Learning Framework) ![](https://img.shields.io/github/stars/knodle/knodle?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| Apache-2 |
 | [NER Annotator for Spacy](https://github.com/tecoholic/ner-annotator) | NER Annotator for SpaCy allows you to create training data for creating a custom NER Model with custom tags. ![](https://img.shields.io/github/stars/tecoholic/ner-annotator?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic) | MIT |
 | [NPLM](https://github.com/BatsResearch/nplm) | Noisy Partial Label Model(NPLM) ![](https://img.shields.io/github/stars/BatsResearch/nplm?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| N/A |
+| [Potato](https://github.com/davidjurgens/potato) | An annotation framework with 20+ tempaltes, editable UI, quality control, data management and even add a surveyflow for crowdourcing ![](https://img.shields.io/github/stars/davidjurgens/potato?color=black&label=%E2%AD%90%EF%B8%8F&style=plastic) | PolyForm Shield |
```
Comment (strickvl) [inline README.md]:
```suggestion
| [Potato](https://github.com/davidjurgens/potato) | An annotation framework with 20+ templates, editable UI, quality control, data management and an option to add a survey for crowdsourcing ![](https://img.shields.io/github/stars/davidjurgens/potato?color=black&label=%E2%AD%90%EF%B8%8F&style=plastic) | PolyForm Shield |
```

**Review by strickvl — CHANGES_REQUESTED** [review]
Thanks for this contribution. Some small changes, but otherwise this looks good to me!

**Comment by Jiaxin-Pei** [comment]
Thanks! I just committed the changes you suggested!

**Review by strickvl — APPROVED** [review]

**Pull request merged**